### PR TITLE
Adds EoY blog post

### PR DESCRIPTION
**Pull request by stockholmux**
### Description

This blog post provides a bit of perspective to summarize the first year of Valkey.
 
### Issues Resolved

n/a

### Check List
- [x] Commits are signed per the DCO using `--signoff`

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: content/blog/2024-12-20-2024-year-of-valkey/index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+date= 2024-12-20 01:01:01
+authors= ["kyledvs"]
++++
+
+The end of the calendar year is a great time to reflect, but for Valkey this particular year-end holds special meaning.
+Think about it: this time in 2023, no one had ever heard the name “Valkey” because, well, it didn’t exist.
+This seems nearly unbelievable given how much has changed in only a few short months.
+
+Now, at the end of 2024, Valkey has had both a minor and major release as well as a few patches.
+Valkey 7.2 primarily introduce the project and new name while carrying over the feature set and performance from before the fork.
```
Comment (madolson) [inline content/blog/2024-12-20-2024-year-of-valkey/index.md:14]:
```suggestion
Valkey 7.2 primarily introduced the project and new name while carrying over the feature set and performance from before the fork.
```

File: content/blog/2024-12-20-2024-year-of-valkey/index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+Valkey 8.0 made substantial internal changes, bringing higher performance through multi-threaded I/O, better memory efficiency from a rewritten main dictionary, and more granular visibility into performance and resource usage.
```
Comment (madolson) [inline content/blog/2024-12-20-2024-year-of-valkey/index.md:15]:
```suggestion
Valkey 8.0 made substantial internal changes, bringing higher performance through multi-threaded I/O, better memory efficiency from a rewritten main dictionary, more granular visibility into performance and resource usage, and enhanced reliability in replication and slot migration.
```

File: content/blog/2024-12-20-2024-year-of-valkey/index.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+## Beginnings
+
+[![YouTube video thumbnail of open source summit north america](images/ossna-thumb.jpg)](https://youtu.be/74Svvu37I_8?si=onLIvlu3X_ncKdh2&t=3055)
+
+Getting started with Valkey has changed substantially over the course of the year.
+When Valkey first launched at Open Source Summit North America, I recall excitedly telling people that you could build from source, get the binary from the website or even use a container.
+Now, you can get Valkey directly from the [package manager in most Linux distributions](https://repology.org/project/valkey/versions) (and more on the way in 2025).
+There are multiple options for containerization and operators to fit your needs.
+And, for those who want to let *others* run Valkey, it is also available as a service on Aiven, AWS, Google Cloud Platform, NetApp Instaclustr, UpCloud, and several more.
```
Comment (madolson) [inline content/blog/2024-12-20-2024-year-of-valkey/index.md:26]:
```suggestion
And, for those who want to let *others* run Valkey, it is also available as a service on Aiven, AWS, Google Cloud Platform, NetApp Instaclustr, UpCloud, and several more.
```
Is that the preferred way to write out GCP? Just odd that it's not written out like AWS.

Comment (madolson) [reply content/blog/2024-12-20-2024-year-of-valkey/index.md:26]: I think this is also a fully managed Valkey, https://elest.io/open-source/valkey.

File: content/blog/2024-12-20-2024-year-of-valkey/index.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+>*We recently adopted Valkey following the unexpected changes with Redis.
+Valkey is a crucial project for the ecosystem, and we’re closely monitoring its progress with the ultimate aim of fully migrating to it.
+We are excited to support the project!*
+
+Similarly, Linux distributions have [widely adopted Valkey in their package managers](https://repology.org/project/valkey/versions).
+The latest version of Fedora, version 41, goes beyond just packaging Valkey: [it obsoletes Redis in lieu of Valkey](https://fedoraproject.org/wiki/Changes/Replace_Redis_With_Valkey#Upgrade/compatibility_impact).
+Functionally, this means that if you upgraded to Fedora 41 and consumed Redis packaged by Fedora, you were automatically migrated to Valkey and all future attempts to install Redis from the package manager gives you Valkey.
+
+[AlmaLinux, “the forever-free, Enterprise OS”](https://almalinux.org/) provides Valkey to it’s users but also is a user of Valkey in their mirroring  system.
```
Comment (madolson) [inline content/blog/2024-12-20-2024-year-of-valkey/index.md:68]:
```suggestion
[AlmaLinux, “the forever-free, Enterprise OS”](https://almalinux.org/) provides Valkey to it’s users but also is a user of Valkey in their mirroring system.
```

**Review by madolson — COMMENTED** [review]
Looks good to me. I can't think of any major event that we're missing.

File: content/blog/2024-12-20-2024-year-of-valkey/index.md (hunk 3 above)
Comment (stockholmux) [reply content/blog/2024-12-20-2024-year-of-valkey/index.md:26]: Okay! I believe they prefer 'Google Cloud Platform' but for symmetry I'll do full name + parenthetical e.g. `Amazon Web Services (AWS)` and `Google Cloud Platform (GCP)`

The listed companies are ones have had formal interactions with the project. I don't believe that elastio has.

**Pull request merged**